examples/inspection/plot_partial_dependence.py

```diff
@@ -240,24 +261,27 @@
 
 import numpy as np
 from mpl_toolkits.mplot3d import Axes3D
+
 fig = plt.figure()
 
-features = ('AveOccup', 'HouseAge')
+features = ("AveOccup", "HouseAge")
 pdp = partial_dependence(
-    est, X_train, features=features, kind='average', grid_resolution=20
+    est, X_train, features=features, kind="average", grid_resolution=20
 )
 XX, YY = np.meshgrid(pdp["values"][0], pdp["values"][1])
 Z = pdp.average[0].T
-ax = Axes3D(fig)
-surf = ax.plot_surface(XX, YY, Z, rstride=1, cstride=1,
-                       cmap=plt.cm.BuPu, edgecolor='k')
+ax = Axes3D(fig, auto_add_to_figure=False)
```

Comment:
`AttributeError: Unknown property auto_add_to_figure`

So it does not seem to be supported in all matplotlib version?